react/ProductGiftImage.tsx

```diff
@@ -0,0 +1,38 @@
+import React, { FC } from 'react'
+import { Image } from 'vtex.store-image'
+
+import { useGift } from './components/ProductGift'
+
+interface Props {
+  maxWidth?: string | number
+  maxHeight?: string | number
+  minWidth?: string | number
+  minHeight?: string | number
+  imageIndex?: number
+}
+
+const DEFAULT_IMAGE_DIMENSIONS = 125
+
+const ProductGiftImage: FC<Props> = ({
+  maxWidth = DEFAULT_IMAGE_DIMENSIONS,
+  maxHeight = DEFAULT_IMAGE_DIMENSIONS,
+  minWidth = DEFAULT_IMAGE_DIMENSIONS,
+  minHeight = DEFAULT_IMAGE_DIMENSIONS,
+  imageIndex = 0,
+}) => {
+  const gift = useGift()
+
+  return (
+    <Image
```

Comment:
No css handle?

Reply:
This `Image` component comes from `vtex.store-image`, which already has CSS handles. I could wrap this with a `div` just to add a handle, but I don't think that's necessary. What do you think?

Reply:
I think it is, how would target it otherwise?

Reply:
Through this:
![Screen Shot 2020-02-21 at 15 59 22](https://user-images.githubusercontent.com/27777263/75063397-9c8c0780-54c3-11ea-85f9-6c64272af47c.png)

Reply:
![Screen Shot 2020-02-21 at 15 58 23](https://user-images.githubusercontent.com/27777263/75063446-b9c0d600-54c3-11ea-8d77-93522033fbad.png)